src/sentry/utils/committers.py

```diff
@@ -246,19 +246,23 @@ def get_event_file_committers(
     if not commits:
         raise Commit.DoesNotExist
 
-    frames = event_frames or ()
+    frames = event_frames or []
+    munged = munged_filename_and_frames(event_platform, frames, "munged_filename")
```

Comment:
Still wondering if it makes sense to call this in `_matching_ownership_rules` or the callers of that method. Just saves us re-munging the frames on every rule the user has set up. Mostly a concern because some users can have 1000s of rows, and this runs in `post_process_group`.

We can just included `munged_filename` in the call to `self.test_frames`, since that will skip any fields that evaluate as falsey anyway

Reply:
Hmm .. I see what you mean. 

Unfortunately, `_matchiner_ownership_rules` takes in the entire `event.data` Mapping, which includes more than just frame data (which is what's currently being used). 

I can refactor the code to:
- deepcopy the entire `event.data` Mapping
- scan through the `data` Mapping, munging all sources of stack frames within
- after munging, delegate to `Matcher.test()` to do the searching on every rule test
- unfortunately, I don't see a way to avoid calling `find_stack_frames` multiple times for the same `data`, unless we wanna refactor `Matcher.test` upstream to do the whole `switch(Matcher.type)` on any callers. 

What do you think?

Reply:
Right... I forgot that we use things other than the frames in that matcher. Tbh I'm not totally sure if it's worth deepcopying the whole thing.

We already have some performance stats around how long ownership takes. Let's just check those after this change goes out and we can decide whether this optimization is worthwhile.

Reply:
Thank goodness .. I was beginning to do the refactor and it would require some big code changes.

I'll keep an eye out on the performance hit this munging stuff will introduce and consider refactoring it down if performance degrades significantly.